### DISCUSSION TITLE
After the component is loaded, all selections are no longer displayed

### DISCUSSION THREAD

**Discussion by DeNIKE666** [livewire/livewire#4452] [1. Help]
Hi I am new to Livewire, started developing an application and faced such a problem that when I open a modal window my selection is hidden and becomes display:none if i specify wire: ignore, but in this case the data is no longer updated, how can i solve this problem? i use wire:model.defer="field" for any field but if i remove it my component is updated and the selection of statuses again display: none

**Comment by joshhanley** [comment]
@DeNIKE666 hmm strange, you shouldn't need to use `wire:ignore` unless you are using a 3rd party modal like bootstrap modal?

Could you share your code which reproduces the issue with as little code as possible so we can investigate it?

Thanks!
